src/Artifacts.jl

```diff
@@ -265,6 +265,20 @@ function unbind_artifact!(artifacts_toml::String, name::String;
     return
 end
 
+if Sys.iswindows()
+    function check_symlinks()
+        tmpdir = mktempdir()
+        fpath = joinpath(tmpdir, "f")
+        touch(fpath)
+        try
+            symlink(fpath, joinpath(tmpdir, "s"))
+            return true
+        catch
+            return false
+        end
```

Comment:
```suggestion
        try
            symlink(fpath, joinpath(tmpdir, "s"))
            return true
        catch
            return false
        finally
            rm(fpath)
        end
```

Reply:
`rm`-ing a dir soon after it's been created often fails on windows because of antivirus. I think it's better to leave this up to the atextit cleanup

Reply:
:scream:

Reply:
You might want to `readlink()` the `fpath` back, just to ensure that it's actually doing what you expect.